src/umpire/strategy/DynamicPool.cpp

```diff
@@ -15,6 +15,7 @@
 #include <cstdlib>
 #include <algorithm>
 #include <sstream>
+#include <new>
```

Comment:
Was this added to get std::bad_alloc{} throw?

Reply:
Yes.

Reply:
Please remove this.